cheri_compressed_cap_common.h

```diff
@@ -326,16 +363,24 @@ static inline void _cc_N(decompress_raw)(_cc_addr_t pesbt, _cc_addr_t cursor, bo
     cdp->cr_perms = (uint32_t)_CC_EXTRACT_FIELD(pesbt, HWPERMS);
     cdp->cr_uperms = (uint32_t)_CC_EXTRACT_FIELD(pesbt, UPERMS);
     cdp->cr_otype = (uint32_t)_CC_EXTRACT_FIELD(pesbt, OTYPE);
+#ifndef IS_MORELLO
     cdp->cr_flags = (uint8_t)_CC_EXTRACT_FIELD(pesbt, FLAGS);
```

Comment:
Does this actually need to be ifdef'ed out or will it just degenerate into extracting 0 bits?

Reply:
The latter. It was only split because for a while I was thinking cheri flags == morello flags and was extracting from cursor here. Now that idea has gone away I will get rid of the ifdef.